src/indra_cogex/client/enrichment/utils.py

```diff
@@ -263,6 +264,32 @@ def get_reactome(*, client: Neo4jClient) -> Dict[Tuple[str, str], Set[str]]:
     return collect_gene_sets(client=client, query=query, cache_file=cache_file)
 
 
+@autoclient(cache=True)
+def get_phenotype_gene_sets(*, client: Neo4jClient) -> Dict[Tuple[str, str], Set[str]]:
+    """Get HPO phenotype gene sets.
+
+    Parameters
+    ----------
+    client :
+        The Neo4j client.
+
+    Returns
+    -------
+    :
+        A dictionary whose keys that are 2-tuples of CURIE and name of each phenotype
+        gene set and whose values are sets of HGNC gene identifiers (as strings)
+    """
+    query = dedent(
+        """\
+        MATCH (s:BioEntity)-[:phenotype_has_gene]-(gene:BioEntity)
+        WHERE s.id STARTS WITH "HP" and gene.id STARTS WITH "hgnc"
+        RETURN s.id, s.name, collect(gene.id);
+    """
+    )
+    logger.info("caching phenotype gene sets with Cypher query: %s", query)
+    return collect_gene_sets(client=client, query=query)
```

Comment:
We need to either add `cache_file` here or handle when `cache_file` is `None` in `collect_gene_sets`. As it is written now, the code in collect_gene_sets will error if `cache_file=None`.

Reply:
Done in c1c68a8

Reply:
I also removed the default none so you can't even call this function without it